### PR TITLE
Edit and deploy readthedocs for LINC CLI

### PR DESCRIPTION
**Pull request by aaronkanzer**
Cc @kabilar -- still in `Draft` mode here -- but this is building well locally -- couple more updates on my side (some text to update from `DANDI Archive` to `LINC Data Platform`) before publishing to the internet.

Screenshot 2024-07-24 at 3 30 38 PM

### CODE REVIEW

File: docs/source/modref/dandiapi.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
-==================
+``lincbrain.dandiapi``
+======================
 
-This module provides functionality for interacting with a Dandi Archive server
+This module provides functionality for interacting with a LINC Data Platform server
 via the REST API.  Interaction begins with the creation of a `DandiAPIClient`
 instance, which can be used to retrieve `RemoteDandiset` objects (representing
-Dandisets on the server) and `BaseRemoteAsset` objects (representing assets
+dataset on the server) and `BaseRemoteAsset` objects (representing assets
```
Comment (kabilar) [inline docs/source/modref/dandiapi.rst:9]:
```suggestion
datasets on the server) and `BaseRemoteAsset` objects (representing assets
```

File: docs/source/conf.py [hunk 2]
```diff
@@ -20,8 +20,8 @@
 import lincbrain
 
 project = "lincbrain"
-copyright = "2021-2023, DANDI Team"
-author = "DANDI Team"
+copyright = "2021-2024, LINC Team"
```
Comment (kabilar) [inline docs/source/conf.py:23]:
```suggestion
copyright = "2023-2024, LINC Team"
```

File: docs/source/modref/dandiapi.rst (hunk 1 above)
Comment (aaronkanzer) [reply docs/source/modref/dandiapi.rst]: Updated!

File: docs/source/conf.py (hunk 2 above)
Comment (aaronkanzer) [reply docs/source/conf.py]: Updated!

**Pull request merged**